src/main/webapp/view.js

```diff
@@ -1,11 +1,57 @@
+/* Used to gather URL parameters. */
```

Comment:
I think most of this stuff belongs in it's own `video.js` file

Reply:
fixed